flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/batch/BatchPhysicalHashAggregate.scala

```diff
@@ -53,7 +53,8 @@ class BatchPhysicalHashAggregate(
     grouping: Array[Int],
     auxGrouping: Array[Int],
     aggCallToAggFunction: Seq[(AggregateCall, UserDefinedFunction)],
-    isMerge: Boolean)
+    isMerge: Boolean,
+    canProjection: Boolean)
```

Comment:
For global hash agg, we don't need this variable, it should always be false.

Reply:
> For global hash agg, we don't need this variable, it should always be false.

Done!